Cargo.toml

```diff
@@ -230,7 +230,7 @@ rpath = false
 [patch.crates-io]
 # If there are dependencies that need patching, they can be listed below.
 
-arrow2 = { git = "https://github.com/jorgecarleitao/arrow2", rev = "f609d0c" }
+arrow2 = { git = "https://github.com/jorgecarleitao/arrow2", rev = "231a6fa" }
 arrow-format = { git = "https://github.com/sundy-li/arrow-format", rev = "c8e11341" }
 parquet2 = { git = "https://github.com/jorgecarleitao/parquet2", rev = "693c265" }
```

Comment:
`parquet2` can be updated to `b0e6545` to fix the bug of can't being able to write decimal values if scale and precision are the same.

Reply:
ok